source/blog.liquid

```diff
@@ -0,0 +1,32 @@
+---
+pagination:
+  data: collections.blog
+  size: 5
+  alias: posts
+  reverse: true
+layout: has_no_sidebars
+title: Sass Blog
+---
+
+Page {{ pagination.pageNumber | plus: 1 }} of {{ pagination.pages.length }}
+<ul class="sl-c-list">
+  {% if pagination.href.previous %}
+    <li>
+      <a href="{{ pagination.href.previous }}">Previous</a>
+    </li>
+  {% endif %}
+  {%- for post in posts -%}
+    <li>
+      <h2>
+        <a href="{{post.url}}">{{ post.data.title }}</a>
+      </h2>
+      <p>Posted {{ post.date | formatBlogDate }} by {{ post.data.author }}</p>
+      <p>{{ post.content | truncatePost }}</p>
```

Comment:
Does including this `<p>` tag cause an empty `<p></p>`?

Reply:
![Screenshot 2023-03-07 at 1 48 35 PM](https://user-images.githubusercontent.com/1581694/223548500-afd3be59-677f-4e77-8ea1-b1c29d47b7d9.jpg)

Reply:
I think it was yes, good spot!